<?xml version="1.0"?>
<launch>
    <arg name="gui" default="false" />

    <node pkg="mrpt_localization" type="mrpt_localization_node" name="mrpt_localization_node" output="screen">
        <param name="ini_file" value="$(find racecar_navigation)/config/pf-localization.ini"/>
        <param name="odom_frame_id" value="/odom"/>
        <param name="global_frame_id" value="/map"/>
        <param name="base_frame_id" value="/base_footprint"/>
        <param name="map_update_skip" value="20"/>
        <param name="particlecloud_update_skip" value="1"/>
        <param name="tf_prefix" value=""/>
        <param name="debug" value="false"/>
        <param name="pose_broadcast" value="true"/>
        <param name="gui_mrpt" value="$(arg gui)"/>
        <param name="sensor_sources" value="/scan"/>

        <remap from="/mrpt_pose" to="/amcl_pose"/>
    </node>
</launch>